webpack/assets/javascripts/react_app/Root/apollo.js

```diff
@@ -0,0 +1,22 @@
+import { ApolloClient, ApolloLink, InMemoryCache, from } from '@apollo/client';
+import { BatchHttpLink } from '@apollo/client/link/batch-http';
+
+const batchLink = new BatchHttpLink({ uri: '/api/graphql' });
```

Comment:
Just to be sure, don't we need to use `foremanUrl` wrapper for every URL?

Reply:
I am not sure either, seems like a we use it in other places so I added it here as well.